string-constants-lib/string-constants/private/danish-string-constants.rkt

```diff
@@ -933,6 +933,54 @@ please adhere to these guidelines:
   (experimental-languages "Eksperimentelle sprog")
   (initial-language-category "Startsprog")
   (no-language-chosen "Intet sprog valgt")
+  (other-languages "Andre sprog")
+
+  (module-language-name "Bestem sprog ud fra kildetekst")
+  (module-language-one-line-summary "#lang-linjen bestemmer det aktuelle sprog")
+  (module-language-auto-text "Automatisk #lang-linje") ;; shows up in the details section of the module language
+  ;; the next four string constants show up in the REPL in DrRacket in the "Racket Language",
+  ;; which is also the "Determine language from source" language. They are put right after the name
+  ;; of the language from the "#lang" line at the beginning of the source file
+  (module-language-repl-no-annotations "")
+  (module-language-repl-debug-annotations ", med debugning")
+  (module-language-repl-debug/profile-annotations ", med debugning og profilering")
+  (module-language-repl-test-annotations ", med testdækning")
+
+  (module-language-#lang-error-more-information "Mere information")
+  (module-language-#lang-flush-cache "Genindlæs")
+  (module-language-#lang-flush-cache-menu-item "Genindlæs #lang-udvidelser")
+
+  ;; for the upper portion of the language dialog
+  (the-racket-language "Sproget Racket")
+  (choose-a-language "Bestem sprog")
+  
+  ;; the next two string constants appear in the
+  ;; language dialog with a list
+  ;; of example languages appearing between them
+  (racket-language-discussion "Begynd dit program med #lang for at udpege den ønskede dialekt. For eksempel:\n\n")
+  (racket-language-discussion-end "\n… og mange flere")
+  
+  ;; the next three string constants are put into a message-box dialog
+  ;; that appears when the user clicks on the example #lang languages
+  ;; in the language dialog. The first one always appears and then either
+  ;; the second or the third appears. The second one has the clicked
+  ;; on #lang line placed into the ~a, and third one has the 
+  ;; current #lang line in the first ~a and the clicked on in the second one.
+  ;; The two comments are separated by a blank line.
+  (racket-dialect-in-buffer-message 
+   "Racket-dialekter bestemmes typisk ved at redigere direkte i bufferen,"
+   " ikke ved at vælge disse muligheder i sprogdialogen.")
+  (racket-dialect-add-new-#lang-line "Skal jeg alligevel tilføje “~a” i definitionsvinduets første linje?")
+  (racket-dialect-replace-#lang-line "Alligevel har du allerede “~a” i din fil; skal jeg erstat det med “~a”?")
```

Comment:
I propose: "Du har allerede “~a” i din fil; skal jeg erstatte det med “~a”?"

Reply:
Hmm ... Github changed the quotation marks inside the string.

Reply:
I see what you meant :)